packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -224,4 +253,467 @@ class _FlutterTestRunnerImpl implements FlutterTestRunner {
+      for (final String plainName in plainNames)
+        ...<String>['--plain-name', plainName],
+      if (randomSeed != null)
+        '--test-randomize-ordering-seed=$randomSeed',
+      if (tags != null)
+        ...<String>['--tags', tags],
+      if (excludeTags != null)
+        ...<String>['--exclude-tags', excludeTags],
+      if (runSkipped)
+        '--run-skipped',
+      if (totalShards != null)
+        '--total-shards=$totalShards',
+      if (shardIndex != null)
+        '--shard-index=$shardIndex',
+      '--chain-stack-traces',
+    ];
+
+    _generateChildTestIsolateSpawnerSourceFile(
+      testFiles,
+      packageTestArgs: packageTestArgs,
+      autoUpdateGoldenFiles: updateGoldens,
+      childTestIsolateSpawnerSourceFile: childTestIsolateSpawnerSourceFile,
+      childTestIsolateSpawnerDillFile: childTestIsolateSpawnerDillFile,
+    );
+
+    _generateRootTestIsolateSpawnerSourceFile(
+      childTestIsolateSpawnerSourceFile: childTestIsolateSpawnerSourceFile,
+      childTestIsolateSpawnerDillFile: childTestIsolateSpawnerDillFile,
+      rootTestIsolateSpawnerSourceFile: rootTestIsolateSpawnerSourceFile,
+    );
+
+    await _compileFile(
+      debuggingOptions: debuggingOptions,
+      sourceFile: childTestIsolateSpawnerSourceFile,
+      outputDillFile: childTestIsolateSpawnerDillFile,
+      testTimeRecorder: testTimeRecorder,
+    );
+
+    await _compileFile(
+      debuggingOptions: debuggingOptions,
+      sourceFile: rootTestIsolateSpawnerSourceFile,
+      outputDillFile: rootTestIsolateSpawnerDillFile,
+      testTimeRecorder: testTimeRecorder,
+    );
+
+    final List<String> command = <String>[
+      globals.artifacts!.getArtifactPath(Artifact.flutterTester),
+      '--disable-vm-service',
+      if (icudtlPath != null) '--icu-data-file-path=$icudtlPath',
+      '--enable-checked-mode',
+      '--verify-entry-points',
+      '--enable-software-rendering',
+      '--skia-deterministic-rendering',
+      if (debuggingOptions.enableDartProfiling)
+        '--enable-dart-profiling',
+      '--non-interactive',
+      '--use-test-fonts',
+      '--disable-asset-fonts',
+      '--packages=${debuggingOptions.buildInfo.packagesPath}',
+      if (testAssetDirectory != null)
+        '--flutter-assets-dir=$testAssetDirectory',
+      if (debuggingOptions.nullAssertions)
+        '--dart-flags=--null_assertions',
+      ...debuggingOptions.dartEntrypointArgs,
+      rootTestIsolateSpawnerDillFile.absolute.path
+    ];
+
+    // If the FLUTTER_TEST environment variable has been set, then pass it on
+    // for package:flutter_test to handle the value.
+    //
+    // If FLUTTER_TEST has not been set, assume from this context that this
+    // call was invoked by the command 'flutter test'.
+    final String flutterTest = globals.platform.environment.containsKey('FLUTTER_TEST')
+        ? globals.platform.environment['FLUTTER_TEST']!
+        : 'true';
+    final Map<String, String> environment = <String, String>{
```

Comment:
Do I need to add an entry for `'FONTCONFIG_FILE'` here like `flutter_tester_device.dart` does?

https://github.com/flutter/flutter/blob/41cc84c0f10931105f0a403b12213c47f75d73f8/packages/flutter_tools/lib/src/test/flutter_tester_device.dart#L143

Reply:
I've added an entry for `'FONTCONFIG_FILE'`.